exercises/concept/amusement-park/.docs/introduction.md

```diff
@@ -17,14 +17,14 @@ class Repeater
   end
 
   def repeat
-    # Outputs the value of the instance variable
-    print @name
+    # Return the value of the instance variable
```

Comment:
We don't mention that we are returning `@heard` (though we are really returning `something`) on line 14, in addition to saving the reference to `something`.  Should we?  It is an early exercise, and may be worthwhile being consistent.  Definitely a different commit though, as it is a different concern.

Generally, though, we would not comment the code.  Instead, talk about the code in prose.  This is an instructional document, so flexibility for learning and explaining sake though.